public/app/plugins/datasource/jaeger/_importedDependencies/types/trace.ts

```diff
@@ -31,6 +31,7 @@ export type TraceLink = {
 export type TraceLog = {
   timestamp: number;
   fields: TraceKeyValuePair[];
+  name?: string;
```

Comment:
`export type TraceLog` is duplicated in a few places. Should we keep it in sync?

Reply:
I think we should. I think you could put it in grafana/data. I wanted to say that you could put it in grafana-o11y-ds-frontend packge, but TraceView is in grafana core and we ideally don't want grafana to have n grafana-o11y-ds-frontend dependency.

Reply:
I just noticed that it is already in grafana/data, so we could just import it from there.

Reply:
Good point! I'll remove it from types/trace.ts but keep other in step 👍

Reply:
Using grafana/data types adds a lot of noise to this PR so I'll do to as a follow up (preview: https://github.com/grafana/grafana/pull/91835/files)